src/Agent/NewRelic/Agent/Core/AgentInstallConfiguration.cs

```diff
@@ -200,19 +200,21 @@ private static string GetNewRelicLogLevel()
 
         public static AgentInfo GetAgentInfo()
         {
-            var agentInfoPath = $@"{NewRelicHome}\agentinfo.json";
+            var agentInfoPath = Path.Combine(NewRelicHome, "agentinfo.json");
+
             if (File.Exists(agentInfoPath))
             {
                 try
                 {
                     return JsonConvert.DeserializeObject<AgentInfo>(File.ReadAllText(agentInfoPath));
                 }
-                catch
+                catch (Exception e)
                 {
-                    // Fail silently
+                    Log.Debug(e, $"Could not deserialize agent info from {agentInfoPath}.");
                 }
             }
 
+            Log.Debug($"Could not get agent info from {agentInfoPath}. File does not exist.");
```

Comment:
Won't we end up logging this if the file does exist but there was a deserialization error?  Should it be in an `else` clause for the `if (File.Exists)...`?

Reply:
Yep, good catch. I've pushed a change.